common/ta_errors.h

```diff
@@ -255,6 +256,14 @@ typedef enum {
   SC_ENDPOINT_DNS_RESOLVE_ERROR = 0x0B | SC_MODULE_ENDPOINT | SC_SEVERITY_FATAL,
   /**< Failed to resolve the domain name address */
 
+  // Crypto module
+  SC_CRYPTO_RAND_INIT = 0x01 | SC_MODULE_CRYPTO | SC_SEVERITY_FATAL,
+  /**< Failed to generate random number generator */
+  SC_CRYPTO_GEN_PKEY = 0x02 | SC_MODULE_CRYPTO | SC_SEVERITY_FATAL,
+  /**< Failed to generate ECDH public key */
+  SC_CRYPTO_COMPUTE_SECRET = 0x03 | SC_MODULE_CRYPTO | SC_SEVERITY_FATAL,
```

Comment:
There is no negative meaning in `SC_CRYPTO_COMPUTE_SECRET`.

Reply:
I have described the negative situation in the message. Or I should rename it as `SC_CRYPTO_WRONG_SECRET`